code/modules/station_goals/bsa.dm

```diff
@@ -115,9 +115,12 @@
 	var/cannon_direction = WEST
 	var/static/image/top_layer = null
 	var/ex_power = 3
-	var/power_used_per_shot = 2000000 //enough to kil standard apc - todo : make this use wires instead and scale explosion power with it
+	/// Amount of energy required to reload the BSA (Joules)
+	var/energy_used_per_shot = 2000000 //enough to kil standard apc - todo : make this use wires instead and scale explosion power with it
```

Comment:
Should this be specified in terms of KJ?

Reply:
MJ, but yes you are correct.

Reply:
done